Add troubleshooting steps for expo logs (#21125)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/expo.md

DdSdkReactNative.initialize(config);
```

## Troubleshooting

### App produces a lot of /logs RUM Resources

> When Resource tracking is enabled and SDK verbosity is `DEBUG`, each RUM Resource will trigger a `/logs` call to the Expo packager to print the log, which will itself create a new RUM resource, etc..

Reviewer: ```suggestion
When Resource tracking is enabled and SDK verbosity is set to `DEBUG`, each RUM Resource will trigger a `/logs` call to the Expo packager to print the log, which will create a new RUM resource.
```

> As we already filter out the most common patterns of Expo packager host URL, so you might not encounter this error in most situations.

Reviewer: ```suggestion
We already filter out the most common patterns of Expo packager host URL, therefore, you may not encounter this error in most situations.
```

> If you do, add the following RUM Resource mapper to filter out the calls:

Reviewer: ```suggestion
If this error occurs, add the following RUM Resource mapper to filter out the calls:
```
